app/router.js

```diff
@@ -72,6 +73,8 @@ module.exports = app => {
     saveQuestionGroup,
   )
 
+  app.post(`/:assessmentUuid/episode/:episodeUuid/referencedata/filtered`, fetchFilteredReferenceData)
```

Comment:
hey Lucas!!, without knowing anything about what you are trying to do, it looks good. 

just wanted to check if `:assessmentUuid` path param is the same as the `:assessmentId` on the endpoint on line 78, i think could be good to rename it to assessmentId to keep the endpoints definitions consistent if it is the same Id they are expecting.

Reply:
Good shout - you're right!, I'll update `assessmentUuid` to `assessmentId` and do the same for `episodeUuid` 👍